data/setup.js

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs').promises;
+
+module.exports = (pool) => {
+  return fs
+    .readFile(`${__dirname}/../sql/setup.sql`, { encoding: 'utf-8' })
```

Comment:
You won't need this line unless you plan on implementing a static front end hosted in the public folder